packages/sanity/src/core/store/_legacy/history/history/TimelineController.ts

```diff
@@ -269,9 +271,10 @@ export class TimelineController {
       queryParams += `&toTransaction=${tid}`
     }
 
-    const url = `/data/history/${dataset}/transactions/${publishedId},${draftId}?${queryParams}`
-
-    const stream = await getJsonStream(this.client.getUrl(url), token)
+    const transactionsUrl = this.client.getUrl(
+      `/data/history/${clientConfig.dataset}/transactions/${publishedId},${draftId}?${queryParams}`
+    )
+    const stream = await getJsonStream(transactionsUrl, clientConfig.token)
```

Comment:
Here too, nothing wrong with the change, but I also cant see the problem with the original code. Just puzzling finding it in the middle of this massive effort ;)

Reply:
This was the other case where `@microsoft/api-extractor` tripped. Thankfully there were only two cases.